File: Python/conditionals/overweight.py
# coding=utf8
# Declaração
e = 0
m = 0

# Entrada
p = float (input("Informe o peso: "))

# Processamento
if p > 50:
    e = p - 50
    m = e * 4
    # Saída
    print("Excesso: {0:.2f}kg".format(e))
    print("Multa: R${0:.2f}".format(m))
else:
    e = 0
    m = 0
    # Saída
    print("Excesso: {0:.2f}kg".format(e))
    print("Multa: R${0:.2f}".format(m))

File: Java/conditionals/overweight.java
package br.com.geekuniversity.secao06;

import java.util.Scanner;

public class Exercicio05 {

	public static void main(String[] args) {
		//Declaração
		float e, m, p;
		Scanner teclado = new Scanner(System.in);
		
		//Entrada
		System.out.println("Digite o peso: ");
		p = teclado.nextFloat();
		
		//Processamento
		if (p > 50) {
			e = p - 50;
			m = (float)e * (float)4;
			//Saída
			System.out.println("Excesso: " + e + "kg");
			System.out.printf("Valor da multa: R$%.2f", m);
		} else {
			e = 0;
			m = 0;
			//Saída
			System.out.println("Excesso: " + e + "kg");
			System.out.printf("Valor da multa: %.2f", m);
		}

		teclado.close();
	}

}
